posts/jhipster-5-release.md

```diff
@@ -0,0 +1,218 @@
+---
+authors:
+- Kenneth Hegeland, Theo Lebrun, Raphael Brugier
+tags:
+- Brace yourself, JHipster 5 is out!
+date: 2018-06-23T15:43:55.000Z
+title: "JHipster 5"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/06/JHipster-5-banner.png
+---
+
+JHipster 5 has just been released, and we thought it would be nice to write a post compiling all the new features. Throughout this post, we will be linking to the official docs, which provide some more detailed explanations of how to use these new features.
+
+With this new release, we think that beginners and experts alike will be excited. We have new technologies being introduced, upgraded versions, and quality of life improvements. Let’s begin with front-end changes.
+
+
+# Frontend changes
+
+## React support
+
+Frontend technologies move fast, and it’s at times difficult to keep up with what is currently the most popular framework. 
+Largely, in the past, Angular has dominated as one of the most popular front-end frameworks, but recently, popularity has been shifting towards React. 
```

Comment:
Paragraphs should be on one line, e.g:

```
A. B. C.
```

not:

```
A.
B.
C.
```

(This applies to the whole post)